js/src/components/wistia-video/index.scss

```diff
@@ -0,0 +1,43 @@
+.wistia-responsive-padding {
```

Comment:
❔ Wondering why it's named with `-padding` postfix?

Reply:
It was copied from the Wistia's sample code 😅. Updated in 2c31424, thanks!